### PR TITLE
Detect flips/armips on unix

### PR DESCRIPTION
**Pull request by Miepee**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -4,8 +4,16 @@ OUT := m4rs
...
 
-FLIPS := ./tools/flips.exe
-AS := ./tools/armips-a8d71f0.exe
+ifeq ($(OS),Windows_NT)
+	FLIPS_TMP := ./tools/flips.exe
+	AS_TMP := ./tools/armips-a8d71f0.exe
+else
+	# Assume unix just has it in PATH
+	FLIPS_TMP := flips
+	AS_TMP := armips
```
Comment (AntyMew) [inline Makefile:13]: `endif` is missing

File: Makefile [hunk 2]
```diff
@@ -4,8 +4,16 @@ OUT := m4rs
...
+
+FLIPS := FLIPS_TMP
+AS := AS_TMP
```
Comment (AntyMew) [inline Makefile:16]: intermediary variables are not necessary, make does not have a concept of scope unless they're defined in a target

**Review by AntyMew — CHANGES_REQUESTED** [review]

File: Makefile (hunk 2 above)
Comment (Miepee) [reply Makefile:16]: Ah, my thought was that if you'd want to do something like `make AS=/mie/pee/flips`. but that probably still works.

Comment (AntyMew) [reply Makefile]: that can be done with `?=` instead

**Review by AntyMew — APPROVED** [review]

**Comment by AntyMew** [comment]
Merging as-is. We can probably improve this build process in the future, but we can work with this for now

**Pull request merged**